resources/views/torrent/edit.blade.php

```diff
@@ -503,6 +503,19 @@ class="form__text"
                         </p>
                     </div>
                 </div>
+                <!-- Porn category extra IDs -->
```

Comment:
These shouldn't be here. They should be moved aboce with rest of meta inputs and use alpine logic like others to only show if category selected is porn_meta